src/Generators/DurableTaskSourceGenerator.cs

```diff
@@ -57,14 +64,15 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 .Where(static func => func != null)!;
 
             // Collect all results and check if Durable Functions is referenced
-            IncrementalValueProvider<(Compilation, ImmutableArray<DurableTaskTypeInfo>, ImmutableArray<DurableFunction>)> compilationAndTasks =
+            IncrementalValueProvider<(Compilation, ImmutableArray<DurableTaskTypeInfo>, ImmutableArray<DurableEventTypeInfo>, ImmutableArray<DurableFunction>)> compilationAndTasks =
                 durableTaskAttributes.Collect()
+                    .Combine(durableEventAttributes.Collect())
                     .Combine(durableFunctions.Collect())
                     .Combine(context.CompilationProvider)
-                    .Select((x, _) => (x.Right, x.Left.Left, x.Left.Right));
+                    .Select((x, _) => (x.Right, x.Left.Left.Left, x.Left.Left.Right, x.Left.Right));
```

Comment:
Can you explain what this is? It just looks like nonsense at this point 😅

Reply:
i know lol, its roslyn api that combines collections in tuples, basically we adds the eventtypeinfo collection and rest stay the same to the provider required for generating the source. below explains it well.

<img width="740" height="518" alt="image" src="https://github.com/user-attachments/assets/1436b675-055b-47f4-b3c8-5a5b6d14c8b0" />